common/app/templates/inlineJS/blocking/curlConfig.scala.js

```diff
@@ -65,7 +65,8 @@ window.curlConfig = {
             'bootstraps/enhanced/media/video-player':   'bootstraps/enhanced/media/video-player-dev.js',
             videojs:                                    'components/video.js/video.js',
             'videojs-contrib-ads':                      'components/videojs-contrib-ads/videojs.ads.js',
-            videojsembed:                               'components/videojs-embed/videojs.embed.js',
+            videojsembed:                               'bootstraps/enhanced/media/videojsembed-dev.js',
+            videojsembeddev:                            'components/videojs-embed/videojs.embed.js',
```

Comment:
Yip, but that's the way we've chosen to do it.
As mentioned this is a hackfix to get our tracking back.